xml/System.Security.Cryptography/IncrementalHash.xml

```diff
@@ -188,11 +205,17 @@
         <Parameter Name="key" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="key">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hashAlgorithm">The name of the hash algorithm to perform within the HMAC.</param>
+        <param name="key">The secret key for the HMAC. The key can be of any length, but a key longer than the output size         of the specified hash algorithm will be hashed to derive a correctly-sized key. Therefore,         the recommended size of the secret key is the output size of the specified hash algorithm.</param>
+        <summary>Creates an <see cref="T:System.Security.Cryptography.IncrementalHash" /> for the Hash-based Message Authentication Code (HMAC)         algorithm using the specified hash algorithm and key.</summary>
+        <returns>An instance of the <see cref="T:System.Security.Cryptography.IncrementalHash" /> class ready to compute the specified hash algorithm.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Unless we have remarks to add, it's fine to leave as-is. They don't show up on the site. I can clean them up if you prefer. But I'd still keep the remarks tag because:
1. If contributors want to add remarks, they know where to put them.
2. The reflection tooling would add the tags back